frontend/src/components/table/InterpreterSearchTable.tsx

```diff
@@ -24,6 +25,9 @@ export default function DirectoryTable({
   const [interpreter, setInterpreter] = useState()
   const [view, setView] = useState('list')
 
+  // Remove "non-active" interpreters
+  data = data.filter(lang => lang.contractExtension || false)
```

Comment:
why `|| false`?

Reply:
also I feel this should be done on the back end for this view, thoughts? was it just extra work to add a new filter query?

Reply:
`|| false` is because I don't come from a dynamic language background so "truthiness" makes me uncomfortable

Reply:
Tuue it should be on the back end ideally, but so should sorting + pagination. I figured this was good enough for now, given that there's not a huge amount of traffic to optimise away.